lightaqonly.lua

```diff
@@ -0,0 +1,34 @@
+-- Changes heavy aquifers to light pre embark
+local help = [====[
+
+lightaqonly
+======
```

Comment:
I think Sphinx is complaining about this line not matching the length of the script name
```suggestion
===========
```

Reply:
Thanks. I'd never have figured out the weird doc complaints looking like doc generation refused to be started had anything to do with lines not being of the desired length!

Reply:
I usually find the GitHub output a bit easier to navigate than Travis:
![image](https://user-images.githubusercontent.com/3719547/75700661-706d4500-5c80-11ea-8814-5143e9306bd9.png)